api/app/reviews/api.py

```diff
@@ -396,11 +416,11 @@ def get(self):
         limit = args['limit']
         sort_column = args['sort_column']
 
-        reviewer = db.session.query(AppUser).get(user_id).is_reviewer(event_id)
+        reviewer = review_repository.get_reviewer(event_id, user_id)
         if not reviewer:
             return FORBIDDEN
 
-        form_id = db.session.query(ApplicationForm.id).filter_by(event_id = event_id).first()[0]
+        form_id = review_repository.get_form_id(event_id)

```

Comment:
I realise you're just replacing the existing code, but it would be better to update the `get_review_history` function and associated query on review_repository to accept an `event_id` rather than a `form_id`, then this additional step of getting the form_id won't be necessary.

Reply:
I updated this, and there are no new errors in my test and the following test is still passing:
- `test_review_history_returned (app.reviews.tests.ReviewsApiTest) ... ok`

I'm not marking as resolved yet, because I still need to fix the removal of `review_responses` and reassess the tests